src/Mercury.ts

```diff
@@ -81,13 +81,6 @@ export type MercuryAuth =
 	| IMercuryAuthSkill
 	| IMercuryAuthUsernamePassword
 
-export interface IMercuryConnectOptions {
```

Comment:
I could not find this being used anywhere, I imagine it'll come back post auto-loaders when this library gets type on construction.